### PR TITLE
Add GetDisplayName() method

### PR DESCRIPTION
**Pull request by MTRNord**
This PR adds the missing GetDisplayName() method.

( Thanks to @t3chguy for the snippet )

### CODE REVIEW

**Comment by t3chguy** [comment]
might be nice to have a helper of osmething like `GetOwnDisplayName()`

File: responses.go [hunk 1]
```diff
@@ -90,6 +90,11 @@ type RespUserInteractive struct {
 	Error     string                 `json:"error"`
 }
 
+// RespUserDisplayName is the JSON response for https://matrix.org/docs/spec/client_server/r0.2.0.html#get-matrix-client-r0-profile-userid-displayname
```
Comment (t3chguy) [inline responses.go]: I'm gonna complain about the placement of this, you're splitting the above struct from its methods below

Comment (MTRNord) [inline responses.go]: Oh didn't saw that method :)

Comment (MTRNord) [inline responses.go]: Fixed

**Comment by Kegsay** [comment]
LGTM

**Comment by ptman** [comment]
So is #36 done?

**Pull request merged**